config/schedule.rb

```diff
@@ -1,24 +1,15 @@
 set :output, "#{ path }/log/daemons.log"
 
-# REVIEW: get times from Settings
-#  require File.expand_path(File.join(File.dirname(__FILE__), '..', 'config', 'environment'))
-#  require 'rubygems'
-#  every ENV['synch_every_minutes'].to_i.minutes do
+if :rails_env == 'production'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.